app/(afterLogin)/(feedback)/feedback/settings/_components/recognition/allRecognition.tsx

```
@@ -195,4 +195,4 @@ const AllRecognition: React.FC<PropsData> = ({ data, all = false }) => {
   );
 };
 
-export default AllRecognition;
+export default AllRecognition;
```

ESLint

[error] Insert `⏎`

(prettier/prettier)